server/cdn-purging/sendPurgeRequestToFastly.ts

```diff
@@ -0,0 +1,35 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+export default function sendPurgeRequestToFastly(surrogateKeys: string[]) {
+  if (!surrogateKeys) {
```

Comment:
-1: surrogateKeys is not allowed null/undefined in the parameter type, add `?` or remove check.